fix: update links to new docs site (#182)
nuxt/starter: README.md

# Nuxt 3 Minimal Starter

> Look at the [nuxt 3 documentation](https://nuxt.com/) to learn more.

Reviewer: ```suggestion
Look at the [Nuxt 3 documentation](https://nuxt.com/docs/) to learn more.
```

> Look at the [Nuxt 3 documentation](https://nuxt.com/docs/) to learn more.

Reviewer: ```suggestion
Look at the [Nuxt 3 documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
```

npm run preview
```

> Checkout the [deployment documentation](https://nuxt.com/docs/getting-started/deployment) for more information.

Reviewer: ```suggestion
Check out the [deployment documentation](https://nuxt.com/docs/getting-started/deployment) for more information.
```
